echo_server.py

```diff
@@ -4,16 +4,30 @@
 
 
 def server_socket_function():
-    server_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP)
+    server_socket = socket.socket(socket.AF_INET,
+                                  socket.SOCK_STREAM,
+                                  socket.IPPROTO_IP)
     server_socket.bind(('127.0.0.1', 50000))
     server_socket.listen(1)
 
     try:
         while True:
             conn, addr = server_socket.accept()
-            message = conn.recv(32)
-            if message:
-                conn.sendall("I recieved your message. Stop talking to me. You are annoying.")
+
+            recieve_total = ""
+            buffersize = 32
+            finished = 0
+            while not finished:
+                recieve = conn.recv(buffersize)
+                if len(recieve) < buffersize:
+                    recieve_total += recieve
+                    finished = 1
+                else:
+                    recieve_total += recieve
```

Comment:
this is a needless doubling of this line of code.  You don't need to do this both here and on line 23 above.